src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+  action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+  method="post"
+  onsubmit="onSubmit()"
+ >
+      <!-- Personal info block -->
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:
+          <input 
+          type="text" 
+          name="surname" 
+          id="surname"
+          autocomplete="off"
+          minlength="2"
+          maxlength="20"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="surname" 
            id="surname"
            autocomplete="off"
            minlength="2"
            maxlength="20"
          >
```
Давай пофіксимо скрізь відступи. Можна почитати про це тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation